src/cli/argument-parser.ts

```diff
@@ -124,13 +124,14 @@ export default class CLIArgumentParser {
             .option('--video <path>', ' record videos of test runs')
             .option('--video-options <option=value[,...]>', 'specify video recording options')
             .option('--video-encoding-options <option=value[,...]>', 'specify encoding options')
-            .option('--disable-page-reloads', 'disable page reloads between tests')
             .option('--dev', 'enables mechanisms to log and diagnose errors')
             .option('--qr-code', 'outputs QR-code that repeats URLs used to connect the remote browsers')
             .option('--sf, --stop-on-first-fail', 'stop an entire test run if any test fails')
             .option('--ts-config-path <path>', 'use a custom TypeScript configuration file and specify its location')
             .option('--cs, --client-scripts <paths>', 'inject scripts into tested pages', this._parseList, [])
             .option('--disable-page-caching', 'disable page caching during test execution')
+            .option('--disable-page-reloads', 'disable page reloads between tests')
+            .option('--disable-screenshots', 'disable taking screenshots')
```

Comment:
Please also change this to just:

```js
            .option('--disable-screenshots', 'disable screenshots')
```

Reply:
And also: `set the amount of time` -> `specify the time`

```js
            .option('--selector-timeout <ms>', 'specify the time within which selectors make attempts to obtain a node to be returned')
            .option('--assertion-timeout <ms>', 'specify the time within which assertion should pass')
            .option('--page-load-timeout <ms>', 'specify the time within which TestCafe waits for the `window.load` event to fire on page load before proceeding to the next test action')
```

Reply:
And... the description for `-e` is printed on the same line:

![image](https://user-images.githubusercontent.com/11459924/66660242-8b95d480-ec4d-11e9-94d7-7a851dc08250.png)

Reply:
And an extra space in the `--video` description
```js
            .option('--video <path>', 'record videos of test runs')
```